source/initial_temperature/world_builder.cc

```diff
@@ -21,7 +21,9 @@
 #include <aspect/global.h>
 
 #ifdef ASPECT_WITH_WORLD_BUILDER
-#include <world_builder/config.h>
+#  include <world_builder/config.h>
+#endif
+
```

Comment:
Shouldn't the other headers also be inside the ifdef section, especially world.h?